### PR TITLE
Update czech translation

### PR DESCRIPTION
**Pull request by Tajnymag**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [x] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [ ] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: resources/language/resource.language.cs_cz/strings.po [hunk 1]
```diff
@@ -1022,54 +1022,54 @@ msgstr "Podle jazyka titulků"
...
 
 # Unused 30502 to 30599
 msgctxt "#30600"
 msgid "ESN / Widevine settings"
-msgstr ""
+msgstr "Nastavení ESN / Widevine"
 
 msgctxt "#30601"
 msgid "ESN:"
 msgstr ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cz/strings.po:1043]: this is missing please add

**Review by CastagnaIT — CHANGES_REQUESTED** [review]
thanks for your new contrib, only a minor thing to fix.
i tag our current languange maintainer @otava5 to have a feedback,
in case of no answers will be accepted as is

**Comment by otava5** [comment]
The changes are fine. Specify a new CZ translation maintainer. Thank you

**Comment by CastagnaIT** [comment]
Fine thanks

**Pull request merged**